daemon/daemon/k8s_watcher.go

```diff
@@ -266,6 +266,41 @@ func (d *Daemon) syncLB() {
 			continue
 		}
 
+		isSvcIPv4 := svcInfo.IP.To4() != nil
+		if isSvcIPv4 && !d.conf.IPv4Enabled {
+			log.Warningf("Received an IPv4 kubernetes service but IPv4 is"+
+				"disabled in the cilium daemon. Ignoring service %+v", svc)
+			continue
+		}
+
+		if isSvcIPv4 {
+			isEPsIPv6 := false
+			for epIP := range se.IPs {
```

Comment:
Sorry, that code changed in the last commit. But still applies and I've change it under `areIPsConsistent` function